core/node/options_network.go

```diff
@@ -22,6 +22,8 @@ type OptionsNetwork struct {
 	Testnet  bool
 	Localnet bool
 
+	IdentityCheck bool
```

Comment:
I would like these options to have `experiment` prefix or suffix. Because we will be deleting them later

Reply:
Agree, fixed it.

Reply:
👍 absolutely